### PR TITLE
Consolidating Querybuilder and reorganizing file structure

### PR DESCRIPTION
**Pull request by ankurp**

### CODE REVIEW

**Comment by ankurp** [comment]
I noticed currently I cannot run the app locally since it is not using the environment specific variables. Has something changed for me to run this locally?

File: server/watson-discovery-service.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ const vcapServices = require('vcap_services');
 const discoveryCredentials = vcapServices.getCredentials('discovery');
 
 const discovery = new DiscoveryV1({
-  username: discoveryCredentials.username,
-  password: discoveryCredentials.password,
+  username: process.env.DISCOVERY_SERVICE_USERNAME || discoveryCredentials.username,
```
Comment (ankurp) [inline server/watson-discovery-service.js]: Noticed it did not use env variables set in `.env` file.

Comment (rhagarty) [inline server/watson-discovery-service.js]: vcapServices handles this. If these credentials are not fond in vcap envs, it grabs them from .env

Comment (rhagarty) [inline server/watson-discovery-service.js]: This worked correctly. vcapServices will find the creds in either the vcap vars or in .env

Comment (rhagarty) [inline server/watson-discovery-service.js]: NOTE that this only works if you rename the vars in .env to DISCOVERY_USERNAME and DISCOVERY_PASSWORD. I have these changes in patch 10.

Comment (ankurp) [inline server/watson-discovery-service.js]: @rhagarty reverted my change for these two lines.

**Comment by rhagarty** [comment]
Thanks. Very nice consolidation.

**Pull request merged**